includes/admin/class-sensei-status.php

```diff
@@ -209,7 +209,8 @@ private function get_templates() {
 		$templates = [];
 
 		$template_dir = Sensei()->plugin_path . 'templates/';
-		$files        = glob( $template_dir . '{*.php,**/*.php}', GLOB_BRACE );
+		$files        = glob( $template_dir . '*.php' );
+		$files       += glob( $template_dir . '**/*.php' );
```

Comment:
Shouldn't it be an `array_concat` or something like that?

Reply:
There's no `array_concat` in PHP, but `+` assumes that function: https://stitcher.io/blog/what-is-array-plus-in-php (yep, I also learned that today..)

Reply:
> There's no `array_concat` in PHP

Oops! haha

> `+`

Nice! But I wonder if that's the same behavior behind the `GLOB_BRACE`. I tried to debug it with the `GLOB_BRACE` and with the `+` and I had a different number of items in the `$files` array. 🤔

Reply:
Excellent catch, @renatho!

I tested it here using a separate script to detect the files on the `templates/` folder and confirmed the behavior you're seeing there on my system (without using the docker container, which doesn't support `GLOB_BRACE`).

Now, I pushed [a195f2b](https://github.com/Automattic/sensei/pull/5315/commits/a195f2b3b41630e2eda6f2ab5c611a7f03599797) to use `array_merge` instead of `+`. Testing it here, this change solves the issue, while also returning the same number of items as if we were using `GLOB_BRACE`. Could you confirm there?

Thanks!

Reply:
Thank you! Worked for me as previously now!